src/FlaUI.WebDriver/Controllers/ElementController.cs

```diff
@@ -178,7 +179,33 @@ public async Task<ActionResult> ElementSendKeys([FromRoute] string sessionId, [F
             {
                 return ElementNotInteractable(elementId);
             }
-            element.AsTextBox().Text = elementSendKeysRequest.Text;
+
+            element.Focus();
+
+            // Warning: Deviation from the spec. https://www.w3.org/TR/webdriver2/#element-send-keys says:
+            //
+            // > Set the text insertion caret using set selection range using current text length for both the start and end parameters.
+            //
+            // In English: "the caret should be placed at the end of the text before sending keys". That doesn't seem to be possible
+            // with UIA, meaning that the text gets inserted at the beginning, which is also WinAppDriver's behavior.
```

Comment:
Shouldn't we document this behavior in our README as a footnote? I'm OK with keeping it consistent with WinAppDriver for now. I would be in favor of a flag "enable WinAppDriver compatibility" in the future when this is possible with UIA.

Reply:
Yep, can add this to the README.

Reply:
Done.